pkg/controllers/gateway/suite_test.go

```diff
@@ -35,10 +35,15 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	clusterv1beta2 "open-cluster-management.io/api/cluster/v1beta1"
```

Comment:
I'm guessing the naming of the var here was an oversight (beta2 vs beta1), hence your change to fix it as ocmclusterv1beta1?

Reply:
Yeah exactly make it a bit more readable to